### PR TITLE
DOCS-5329 Remove unused screenshots from repo; remove unnecessary screenshots from docs

### PR DESCRIPTION
**Pull request by alexisintech**
let's get our repo images cleaned up ! get our repo bundle smaller :~)

### CODE REVIEW

File: docs/authentication/social-connections/oauth.mdx [hunk 1]
```diff
@@ -106,18 +106,18 @@ export async function GET() {
 
 ## Add social connection after sign-up
 
-For each OAuth provider, you can choose whether it will be available during sign-up and sign-in, or if the connection should be made later.
+For each OAuth provider, you can disable the option to sign up and sign in to your application using the provider. This is especially useful for users that want to connect their OAuth account after authentication.
 
-This is especially useful for applications that prefer to connect third-parties after the fact. For example, a Github connection can be made after sign-up if an application wants to read repository data.
+For example, say your application wants to read a user's GitHub repository data but doesn't want to allow the user to authenticate with their GitHub account. The user can sign up with their email and password, or whatever method you choose, and then afterwards, connect their GitHub account to your application.
```
Comment (S3Prototype) [inline docs/authentication/social-connections/oauth.mdx:111]: There's still a screenshot on line 38 of this page. Intentional?

Comment (S3Prototype) [reply docs/authentication/social-connections/oauth.mdx:111]: image

File: docs/deployments/deploy-to-vercel.mdx [hunk 2]
```diff
@@ -19,21 +19,21 @@ Let's walk through the following:
 
 To set up a new Vercel project, start by [going to your Vercel Dashboard](https://vercel.com/dashboard). Here, you'll see a list of your projects or, if you don't have any projects yet, a prompt to create a new project:
 
-![A Vercel dashboard with a "No projects, yet!" message. Buttons to create a new project and browse templates are present.](/docs/images/deployment-guides/vercel-no-new-projects.png)
+![A Vercel dashboard with a "No projects, yet!" message. Buttons to create a new project and browse templates are present.](/docs/images/deployments/vercel-no-new-projects.png)
```
Comment (S3Prototype) [inline docs/deployments/deploy-to-vercel.mdx:22]: I'm skeptical of having screenshots of another product's UI in our docs, as we'd have no idea if these got out of date, but we can leave them for now

**Review by S3Prototype — APPROVED** [review]

File: docs/deployments/deploy-to-vercel.mdx (hunk 2 above)
Comment (alexisintech) [reply docs/deployments/deploy-to-vercel.mdx:22]: yeah we can definitely revisit these guides!

File: docs/authentication/social-connections/oauth.mdx (hunk 1 above)
Comment (alexisintech) [reply docs/authentication/social-connections/oauth.mdx:111]: let me see what I can do about the copy to see if we can indicate it better

Comment (alexisintech) [reply docs/authentication/social-connections/oauth.mdx:111]: good point ! [2cfad17](https://github.com/clerk/clerk-docs/pull/1071/commits/2cfad17a6d45bf493126dc1cb722b727995ee826)

**Pull request merged**